packages/Core/src/WuiProvider.tsx

```diff
@@ -20,12 +24,14 @@ export const WuiProvider: React.FC<WuiProviderProps> = ({
   reactRootId = 'root',
   shouldHideFocusRingOnClick = true,
   theme,
+  usePanda,
   useReset,
 }) => {
   return (
     <ThemeProvider theme={theme}>
       <>
-        {hasGlobalStyle && <GlobalStyle useReset={useReset} />}
+        {hasGlobalStyle && !usePanda && <GlobalStyle useReset={useReset} />}
+        {usePanda && <GlobalStylePanda />}
         {shouldHideFocusRingOnClick ? (
```

Comment:
How did you do if we've got the 2 in same time ?

Reply:
If we have both solutions at the same time, xstyled injects `<GlobalStylePanda />` -> global styles needed by xstyled to work and the rest of our global styles are injected by 🐼

Reply:
cf `packages/Core/src/panda/globalCss.ts`
https://github.com/WTTJ/welcome-ui/pull/2317/files#diff-6aa910a82ab785bb8491cb905dd60bda018bac8869350ac450f3dbe3cecec7ca